backend/src/main/java/com/wootech/dropthecode/domain/TeacherProfile.java

```diff
@@ -29,9 +29,9 @@
     @OneToMany(mappedBy = "teacherProfile", fetch = FetchType.LAZY, cascade = CascadeType.REMOVE)
     private final Set<TeacherSkill> skills = new HashSet<>();
 
-    private final Integer sumReviewCount = 0;
+    private Integer sumReviewCount = 0;
```

Comment:
sumReviewCount가 리뷰를 한 횟수인 것 같은데, 이 경우 int로 두면 0으로 초기화 하지 않아도 기본값이 0으로 들어갈 것 같은데 어떻게 생각하시나요?
```suggestion
    private int sumReviewCount;
```

Reply:
오 좋은데요 적용했습니다.